src/aws-cpp-sdk-core/include/smithy/client/AwsSmithyClient.h

```diff
@@ -222,7 +222,7 @@ namespace client
             auto httpResponseOutcome = MakeRequestSync(request, requestName, method, std::move(endpointCallback));
             return m_serializer->Deserialize(std::move(httpResponseOutcome), GetServiceClientName(), requestName);
         }
-
+        
         Aws::String GeneratePresignedUrl(
```

Comment:
so fan of having on presigner now, only question i really have is why is this in the the Smithy client and the others legcy. I think something that we could talk about is "what the public interface of a pre signer should be". but since this is already here, im open to moving forward without having a answer for that for now